components/actiongroup/stories/actiongroup.stories.js

```diff
@@ -43,16 +46,45 @@ export default {
 			},
 			control: "boolean",
 		},
+		iconOnly: {
+			name: "Icon Only",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Advanced",
```

Comment:
Does `iconOnly` belong in "Advanced", or is it just "Component?" I don't really know or have much of an opinion, just curious. Some other components that have some similar icon controls, like contextual help, tabs, and menu (technically menu item) have it in "Component" or "Content." Just a thought for you!